src/ui/ui_settings/defaults.js

```diff
@@ -341,6 +341,12 @@ export function getDefaultSettings() {
     'context:step': {
       value: 5,
       description: 'The step size to increment or decrement the context size by',
-    }
+    },
+    'context:tieBreakerFields': {
+      value: ['_doc'],
+      description: 'A list of fields to use for tie-breaking between documents ' +
```

Comment:
Could you add that it's a comma delimited list of fields? Even I had to stop for a moment to figure out how to enter multiple fields in the input.